File: app/src/main/java/com/example/influencer/UI/OnBoarding___CountryAndDateSelector/OnBoarding/Fragments/OnboardingFragment3.java
package com.example.influencer.UI.OnBoarding___CountryAndDateSelector.OnBoarding.Fragments;

import android.os.Bundle;

import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.influencer.databinding.FragmentOnboarding3Binding;


public class OnboardingFragment3 extends Fragment {

    private FragmentOnboarding3Binding binding;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        binding = FragmentOnboarding3Binding.inflate(inflater, container, false);
        return binding.getRoot();
    }
}

File: app/src/main/java/com/example/influencer/Features/OnBoarding___CountryAndDateSelector/UI/OnBoarding/Fragments/OnboardingFragment3.kt
package com.example.influencer.Features.OnBoarding___CountryAndDateSelector.UI.OnBoarding.Fragments

import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import com.example.influencer.databinding.FragmentOnboarding3Binding

class OnboardingFragment3 : Fragment() {

    private var _binding: FragmentOnboarding3Binding? = null
    private val binding get() = _binding!!

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        _binding = FragmentOnboarding3Binding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }
}